packages/opentelemetry-instrumentation-http/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentelemetry/instrumentation-http",
-  "version": "0.23.0",
+  "version": "0.23.1",
```

Comment:
please revert version changes here and in version.ts because this is done during release

Reply:
got it!